### ISSUE TITLE
Documentation missing info for manual installation

### ISSUE THREAD

**Issue by Darkentik** [msmhq/msm#396]
Hey,

I've follow your docs under http://msmhq.com/docs/installation.html for manual installation and find some steps that are missing:
Before step 9 (creating jargroup) you have to do:
- create the folder: `mkdir /opt/msm/jars`
- set ownership: `chown minecraft /opt/msm/jars`
Otherwise you got this error:
`root@mcserver01-a:/# msm jargroup create minecraft minecraft
Creating jar group... -bash: /opt/msm/jars/minecraft/target.txt: Datei oder Verzeichnis nicht gefunden
Done.`

Before step 10 (creating server) you have to do:
- create the folder: `mkdir /opt/msm/servers`
- set ownership: `chown minecraft /opt/msm/servers`
Otherwise you got this error:
`root@mcserver01-a:/# msm server create mcserver01-1-16
Creating server directory... mkdir: das Verzeichnis „/opt/msm/servers“ kann nicht angelegt werden: Keine Berechtigung
touch: '/opt/msm/servers/mcserver01-1-16/white-list.txt' kann nicht berührt werden: Datei oder Verzeichnis nicht gefunden
touch: '/opt/msm/servers/mcserver01-1-16/banned-ips.txt' kann nicht berührt werden: Datei oder Verzeichnis nicht gefunden
touch: '/opt/msm/servers/mcserver01-1-16/banned-players.txt' kann nicht berührt werden: Datei oder Verzeichnis nicht gefunden
touch: '/opt/msm/servers/mcserver01-1-16/ops.txt' kann nicht berührt werden: Datei oder Verzeichnis nicht gefunden
touch: '/opt/msm/servers/mcserver01-1-16/server.properties' kann nicht berührt werden: Datei oder Verzeichnis nicht gefunden
mkdir: das Verzeichnis „/opt/msm/servers“ kann nicht angelegt werden: Keine Berechtigung
-bash: /opt/msm/servers/mcserver01-1-16/worldstorage/readme.txt: Datei oder Verzeichnis nicht gefunden
Done.`

If i find some more i will update this here.

Thanks a lot.

Best regards!
Darkentik

**Comment by renderorange** [comment]
Hi Darkentik,

The `msm server create` command should be creating the `servers` and `jars` directories for you if they're not present.  

Manually testing the command creating the directories, I removed the directories then ran the commands you mentioned.  Both were re-created correctly.

```
server_create() {
(...)
                        printf "Creating server directory... "
                        as_user "$SETTINGS_USERNAME" "mkdir -p '$SETTINGS_SERVER_STORAGE_PATH/$1'"
```
```
root@dig /opt/msm # rm -rf servers/
root@dig /opt/msm # ls -lah
total 24K
drwxr-xr-x 4 minecraft minecraft 4.0K Jan  1 17:08 .
drwxr-xr-x 4 root      root      4.0K Sep 26 23:33 ..
-rw------- 1 minecraft minecraft  116 Sep 27 14:20 .bash_history
drwxr-xr-x 3 minecraft minecraft 4.0K Sep 26 23:33 jars
drwxr-xr-x 4 root      root      4.0K Sep 26 23:33 versioning
-rw-r--r-- 1 minecraft minecraft  180 Sep 26 23:33 .wget-hsts
root@dig /opt/msm # msm server create test
Creating server directory... Done.
Downloading latest version... Checking minecraft version JSON... Done.
Existing version "/opt/msm/jars/minecraft/2021-01-01-17-07-18-minecraft_server.1.16.4.jar" was already up to date.
Server "test" is now using "/opt/msm/jars/minecraft/2021-01-01-17-07-18-minecraft_server.1.16.4.jar".
root@dig /opt/msm # ls -lah servers/
total 12K
drwxr-xr-x 3 minecraft minecraft 4.0K Jan  1 17:08 .
drwxr-xr-x 5 minecraft minecraft 4.0K Jan  1 17:08 ..
drwxr-xr-x 3 minecraft minecraft 4.0K Jan  1 17:08 test
```

I suspect the issue you're seeing is due to permissions on the `/opt/msm` directory not being correct to allow the `minecraft` use to create the directories.

Reading through the installation instructions, the commands listed don't set the permissions the same as the installer.  I'm going to create a pull request to update the documentation so the installation instructions are the same.  Then perhaps others won't run into this same issue.

Blaine

**Comment by endofcake** [comment]
Resolved in #413 .
